roles/openshift_facts/library/openshift_facts.py

```diff
@@ -1751,6 +1751,7 @@ def get_defaults(self, roles, deployment_type, deployment_subtype):
                 {"name": "PodFitsPorts"},
                 {"name": "NoDiskConflict"},
                 {"name": "NoVolumeZoneConflict"},
+                {"name": "VolumeZonePredicate"},
```

Comment:
I think you may be right, I was looking at ways to resolve issues with multizone volumes. Ideal fix seems to be in 1.4.